### PR TITLE
Fix tests failing when executed on Windows

### PR DESCRIPTION
**Pull request by jarst**

### CODE REVIEW

File: liquigraph-core/src/main/java/org/liquigraph/core/io/xml/ChangelogParser.java [hunk 1]
```diff
@@ -83,7 +83,7 @@ private void fixUpChangesets(Changelog changelog) {
     }
 
     private String formatErrorMessage(Collection<String> errors) {
-        String separator = "\n\t";
+        String separator =  System.getProperty("line.separator") + "\t";
```
Comment (fbiville) [inline liquigraph-core/src/main/java/org/liquigraph/core/io/xml/ChangelogParser.java]: We're on Java 7, would you mind using http://docs.oracle.com/javase/8/docs/api/java/lang/System.html#lineSeparator-- instead?

**Comment by fbiville** [comment]
Thanks a lot for your efforts :)
I just noted a tiny issue and I'll merge this soon!

File: liquigraph-core/src/main/java/org/liquigraph/core/io/xml/ChangelogParser.java (hunk 1 above)
Comment (jarst) [inline liquigraph-core/src/main/java/org/liquigraph/core/io/xml/ChangelogParser.java]: Sure thing.

**Comment by fbiville** [comment]
Tested on a Windows machine, squashed and merged in 738227d3eae773164ab9827c0457770e273d8103.

Thanks a lot!

**Pull request closed without merging**